lm-feedCore-iOS/lm-feedCore-iOS/Source/Classes/Operations/LMFeedEditPostOperation.swift

```diff
@@ -48,43 +48,43 @@ final class LMFeedEditPostOperation {
         media.forEach { medium in
             switch medium.isVideo {
             case true:
-                let attachmentMeta = AttachmentMeta()
-                    .attachmentUrl(medium.url)
-                    .size(medium.size)
-                    .name(medium.name)
-                    .duration(medium.duration ?? 0)
+                var attachmentMeta = AttachmentMeta.Builder()
+                attachmentMeta = attachmentMeta.attachmentUrl(medium.url)
+                attachmentMeta = attachmentMeta.size(medium.size)
+                attachmentMeta = attachmentMeta.name(medium.name)
+                attachmentMeta = attachmentMeta.duration(medium.duration ?? 0)
                 
                 
                 let attachment = Attachment()
                     .attachmentType(.video)
-                    .attachmentMeta(attachmentMeta)
+                    .attachmentMeta(attachmentMeta.build())
                 
                 attachments.append(attachment)
             case false:
-                let attachmentMeta = AttachmentMeta()
-                    .attachmentUrl(medium.url)
-                    .size(medium.size)
-                    .name(medium.name)
+                var attachmentMeta = AttachmentMeta.Builder()
+                attachmentMeta = attachmentMeta.attachmentUrl(medium.url)
+                attachmentMeta = attachmentMeta.size(medium.size)
+                attachmentMeta = attachmentMeta.name(medium.name)
```

Comment:
why we have to do this way?

Reply:
AttachmentMeta was not on Builder pattern, have to shift it to builder pattern.